org.eclipse.xtext.ui.tests/src-gen/org/eclipse/xtext/ui/tests/editor/contentassist/services/Bug377311TestLanguageGrammarAccess.java

```diff
@@ -118,7 +118,7 @@ public ParserRule getRootRule() {
 		return getRootAccess().getRule();
 	}
 	
-	//Child hidden(WS, ID):
+	//Child hidden (WS , ID):
```

Comment:
@ArneDeutsch Another thing that is missing for hidden clauses: The space before the `,` is unexpected.

Reply:
Should be fixed with https://github.com/eclipse/xtext-core/issues/1489